data-structures/4-semesters-of-computer-science.md

```diff
@@ -18,7 +18,7 @@ If you are in the San Francisco Bay Area and you get a SFPL library card, you ge
 ### Specific Things to Learn
 - Things about the things
 - More things about the things
-	- This is a sub-thing about the things
```

Comment:
Please remove any changes not related to this PR.

Reply:
This file should not be here at all. This may help: https://stackoverflow.com/questions/1817766/how-to-revert-to-origins-master-branchs-version-of-file

Reply:
Welcome to the world of git! You can do it 😉